src/hobbits-core/bitcontainer.cpp

```diff
@@ -88,6 +88,10 @@ QString BitContainer::name() const
     return m_name;
 }
 
+qint64 BitContainer::size() const {
```

Comment:
do you think this is really necessary? I see some brevity benefits, but I don't want containers to be easily confused with the BitArrays